### PR TITLE
Add documentation for Analysis messsage IST0132 VirtualServiceHostNotFoundInGateway

### PR DESCRIPTION
**Pull request by zufardhiyaulhaq**
Related Issue: https://github.com/istio/istio/issues/28172
Related Analyzer PR: https://github.com/istio/istio/pull/28486

[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by frankbu** [comment]
The documentation for this message is generated from https://github.com/istio/istio/blob/9567e6f481e42e4c47405d445bdafe9c3cd75502/galley/pkg/config/analysis/msg/messages.yaml#L366.
It will appear in the preliminary docs as soon as the release-1.8 branch is cut.

**Comment by zufardhiyaulhaq** [comment]
@frankbu thanks for the information, we need to modify this README.md then. https://github.com/istio/istio/blob/master/galley/pkg/config/analysis/README.md#7-write-a-user-facing-documentation-page

**Comment by frankbu** [comment]
> @frankbu thanks for the information, we need to modify this README.md then. https://github.com/istio/istio/blob/master/galley/pkg/config/analysis/README.md#7-write-a-user-facing-documentation-page

@zufardhiyaulhaq Yes, thanks. This change came as a surprise to me too, to be honest :)

**Comment by frankbu** [comment]
After looking into this, it seems like the document is optional. If one exists it is used, otherwise just the generated description is used for the docs. So the question is, do we want this one more completely documented? Seems reasonable to me. But, the next question is whether this is for 1.8 or post 1.8. As it stands, the generated part is not available in the 1.8 branch, so it won't be out until after the release, in which case this PR should not merge until then.

**Comment by zufardhiyaulhaq** [comment]
@frankbu we can delay this MR to be merged until 1.8

File: content/en/docs/reference/config/analysis/ist0132/index.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: VirtualServiceHostNotFoundInGateway
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: no
+---
+
+This message occurs when the `host` defined in VirtualService is not found in the Gateway.
```
Comment (frankbu) [inline content/en/docs/reference/config/analysis/ist0132/index.md:8]:
```suggestion
This message occurs when a `host` defined in a VirtualService is not found in the corresponding Gateway.
```

File: content/en/docs/reference/config/analysis/ist0132/index.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Example
+
+You will receive this message:
+
+{{< text plain >}}
+Warning [IST0132] (VirtualService testing-service.default testing.yaml:8) one or more host [wrong.com] defined in VirtualService default/testing-service not found in Gateway istio-system/testing-gateway.
+{{< /text >}}
+
+when your cluster has following VirtualService:
```
Comment (frankbu) [inline content/en/docs/reference/config/analysis/ist0132/index.md:18]:
```suggestion
when your cluster has the following virtual service:

```

File: content/en/docs/reference/config/analysis/ist0132/index.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  servers:
+  - hosts:
+    - testing.com
+    port:
+      name: http
+      number: 80
+      protocol: HTTP
+{{< /text >}}
+
+In this example, VirtualService `testing-service` has host `wrong.com` but it is not found in the gateway `testing-gateway`.
```
Comment (frankbu) [inline content/en/docs/reference/config/analysis/ist0132/index.md:58]:
```suggestion
In this example, virtual service `testing-service` has host `wrong.com` which is is not found in the gateway `testing-gateway`.
```

File: content/en/docs/reference/config/analysis/ist0132/index.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+## How to resolve
+
+- Make sure `hosts` in the VirtualService is exist in the Gateway that related to the VirtualService itself.
```
Comment (frankbu) [inline content/en/docs/reference/config/analysis/ist0132/index.md:62]:
```suggestion
Make sure all `hosts` in a virtual service are included in the `hosts` of gateways that are bound to the virtual service.
```

**Comment by zufardhiyaulhaq** [comment]
@frankbu did you know what is the problem with the CI?
- lint_istio.io show
```
- ./public/docs/reference/config/analysis/ist0132/index.html
  *  65162:18: ERROR: Unexpected end tag : p (line 65162)
```
I cannot find any HTML `p` tags.
- deploy/netlify
```
Building sites … ERROR 2020/11/26 01:25:11 Could not find analysis message for VirtualServiceHostNotFoundInGateway
```
not sure about this one

File: content/en/docs/reference/config/analysis/ist0132/index.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  http:
+  - match:
+    - uri:
+        prefix: /
+    route:
+    - destination:
+        host: ratings
+{{< /text >}}
+
+and with this following Gateway:
```
Comment (frankbu) [inline content/en/docs/reference/config/analysis/ist0132/index.md:40]: @zufardhiyaulhaq I think a missing blank line here is causing your lint error.
```suggestion
and with this following Gateway:

```

**Review by frankbu — APPROVED** [review]

**Pull request merged**